services/IBF-pipeline/pipeline/settings.py

```diff
@@ -36,8 +36,8 @@
             'triggerStatement': "TBD",
             'linkDashboard': os.getenv('DASHBOARD_URL'),
             'linkEAPSOP': "https://adore.ifrc.org/Download.aspx?FileId=357745",
-            'linkVideo': "https://www.youtube-nocookie.com/embed/3hm4IeLP90g",
-            'linkPdf':  "https://unstats.un.org/unsd/demographic/sources/census/wphc/Uganda/UGA-2016-05-23.pdf",
+            'linkVideo': "https://www.youtube.com/watch?v=q_xTy9PpCcE",
```

Comment:
please use the no-cookie version of the youtube links instead of the browser URLs.

`https://www.youtube-nocookie.com/embed/q_xTy9PpCcE` is the correct value for this one.